tests/support/pretest.js

```diff
@@ -110,25 +113,29 @@ function getHarness(tree) {
 }
 
 function modFiles() {
-	return glob(path.join(testPath, '*.html')).then(function(paths) {
+	return glob(path.join(testPath, '**/*.html')).then(function(paths) {
 		return Promise.all(_.map(paths, function(path) {
 			return fs.readFileAsync(path, 'utf8')
-				.then(modFile)
+				.then(function(source) {
+					return modFile(source, path);
+				})
 				.then(function(source) {
 					return fs.writeFileAsync(path, source, 'utf-8');
 				});
 		}));
 	});
 }
 
-function modFile(source) {
+function modFile(source, filePath) {
+	var fileDir = path.dirname(filePath);
+	var pepPath = path.relative(fileDir, path.join(basePath, 'dist', 'pep.js'));
+	var supPath = path.relative(fileDir, path.join(basePath, 'tests', 'support', 'pep_support.js')),
 
 	// Ensure pep.js is the first script loaded on the page
-	source = source.replace(/^\s*(?=<script\b)/im, '\n$&<script src="' + encodeURI(pepPath) + '"></script>\n$&');
+	source = source.replace(/^\s*(?=<script\b|<\/head>)/im, '\n$&<script src="' + encodeURI(pepPath) + '"></script>\n$&');
```

Comment:
One of the iframe HTML files does not have any scripts, so PEP just gets added as the last line of the `<head>` block.